### PR TITLE
Bug template tag updates for 1.1.5, 1.2-preview1, and 1.2-experimental2

### PR DESCRIPTION
**Pull request by gabbybilka**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -35,8 +35,9 @@ body:
       label: NuGet package version
       description: Specify the version of Windows App SDK (or Project Reunion) you're using.
       options:
-        - "1.2.220727.1-experimental1"
-        - "1.1.4"
+        - "1.2.0-preview1"
```
Comment (kythant) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:38]: Shouldn't this be [1.2.220902.1-preview1](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.2.220902.1-preview1)?

Comment (gabbybilka) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: Yeah, we can keep it consistent. Fixed in latest commit.

**Review by kythant — APPROVED** [review]

**Pull request merged**